site/docs/blog/namespace-enforcement.md

```diff
@@ -0,0 +1,5 @@
+# Namespace enforcement in Nessie
+
+* From Nessie version 0.52.3 and onwards, Namespaces must exist before creating the tables.
+<br>[Background](https://github.com/projectnessie/nessie/issues/6244)
+<br>[more info](../../../api/NESSIE-SPEC-2-0.md#200-beta1)
```

Comment:
Should I also add that for existing tables with the implicit namespaces, the new commits will fail after the upgrade due to this enforcement and namespaces must be created explicitly again?

Reply:
Yes, I think it makes sense to mention that.

Reply:
added